### PR TITLE
Doc/vnv/system test

### PR DESCRIPTION
**Pull request by Wililee**
Please read them, 

These are the functional requirement system tests.
Test IDs will be placed at a later date.

image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by dimitritsampiras** [comment]
I think you should categorize these tests under specific "Areas"

File: docs/VnVPlan/VnVPlan.tex [hunk 1]
```diff
@@ -189,33 +189,278 @@
...
+		
+		Requirement(s): R1
+		
+		\item{\textbf{test-id1}}: Adding an exercise to a workout routine.
+
+		Control: Manual
+		
+		Initial State: Application is running and a workout routine is in the process of being created or edited.
+		
+		Input: a user created exercise with the parameters required for the exercise.
```
Comment (bentvelj) [inline docs/VnVPlan/VnVPlan.tex:230]:
```suggestion
		Input: A user created exercise with the parameters required for the exercise.
```

File: docs/VnVPlan/VnVPlan.tex [hunk 2]
```diff
@@ -189,33 +189,278 @@
...
+		
+		Initial State: Application is running and a workout routine is in the process of being created or edited.
+		
+		Input: The number of exercises required to reach the limit of exercises per workout routine. 
+		
+		Output: A notification to the user, before the limiting exercise notifying them of the limit, and preventing them from adding another exercise.
+		
+		Test Case Derivation: The user should be aware of the exercise limit per workout routine and should not be able to exceed it.
+		
+		How test will be performed: The test will be preformed by manually added exercises to a workout routine until the limit. Then upon adding the limiting input, a notification should be expected.
```
Comment (bentvelj) [inline docs/VnVPlan/VnVPlan.tex:268]:
```suggestion
		How test will be performed: The test will be preformed by manually added exercises to a workout routine until the limit. Then upon adding the limiting input, a notification is expected.
```

File: docs/VnVPlan/VnVPlan.tex [hunk 3]
```diff
@@ -189,33 +189,278 @@
...
+		
+		Requirement(s): R3
+		
+		\item{\textbf{test-id1}}: Publicizing a workout routine.
+		
+		Control: Manual
+		
+		Initial State: The application is running and a private workout routine is created.
+		
+		Input: An edit or addition to a workout routine to make is public.
```
Comment (bentvelj) [inline docs/VnVPlan/VnVPlan.tex:326]:
```suggestion
		Input: An edit or addition to a workout routine to make it public.
```

File: docs/VnVPlan/VnVPlan.tex [hunk 4]
```diff
@@ -189,33 +189,278 @@
...
+		
+		Control: Manual
+		
+		Initial State: The application is running and multiple workout routines are created.
+		
+		Input: Search string inputs to view the public workout routines.
+		
+		Output: Public workout routines that match the search criteria should be displayed. 
+		
+		Test Case Derivation: All workout routine that follow match a search, should be display.
```
Comment (bentvelj) [inline docs/VnVPlan/VnVPlan.tex:394]:
```suggestion
		Test Case Derivation: All workout routines that match a search, should be displayed.
```

**Review by bentvelj — CHANGES_REQUESTED** [review]
Some typos but looks fantastic

**Review by bentvelj — APPROVED** [review]
LGTM

**Pull request merged**